leaf-ui/scripts/main.js

```diff
@@ -26,7 +25,10 @@ var reader;
 
 //Properties for both core and simulator.
 //TODO: merge this two arrays in order to make use the same name for all
-var satvalues = {satisfied: 2, partiallysatisfied: 1, partiallydenied: -1, denied: -2, unknown: 4, conflict: 3, none: 0};
+var satvalues = {
```

Comment:
That is why I put that "//TODO" comment, a lot of parts in the code for example: the select initial values are using one type of annotation in the satvalues array and some parts of the code are using the other array with "0000, 0001, etc..." type of annotation. Since they "mean" the same thing it is a good idea refactor and use just one.